documentation/1.0.0/dockerized/scripts/windows/insert_entity_project_data.js

```diff
@@ -0,0 +1,57 @@
+const { MongoClient } = require('mongodb');

```

Comment:
@MallanagoudaB this script is used for inserting data to db for dockerized windows setup?

Reply:
Yes @VISHNUDAS-tunerlabs